crates/hyperqueue/src/client/output/quiet.rs

```diff
@@ -130,6 +131,8 @@ impl Output for Quiet {
     ) {
     }
 
+    fn print_task_ids(&self, _job_task_ids: Vec<(JobId, IntArray)>) {}
```

Comment:
I think that here we could write one ID per line? This mode should really be renamed from quiet to something like `bash-processable` :)

Reply:
I have changed it. if there are 2 or more jobs, it prints also job ids.